### ISSUE TITLE
tools/1to2.js fails to generate proper source

### ISSUE THREAD

**Issue by Gottox** [nodejs/nan#441]
I'm using NAN-1 with https://github.com/Gottox/child_pty. In https://github.com/Gottox/child_pty/issues/7 it came up, that NAN-2 breaks the build process.

I tried to use the 1to2.js script, but it generates wrong constructs, such as:

```diff
-               if(obj->Has(columns))
-                       w->ws_col = obj->Get(columns)->Uint32Value();
-               if(obj->Has(rows))
-                       w->ws_row = obj->Get(rows)->Uint32Value();
+               NanHas(if(obj, columns))
+                       w->ws_col = NanGet(obj, columns)->Uint32Value();
+               NanHas(if(obj, rows))
+                       w->ws_row = NanGet(obj, rows)->Uint32Value();
```

Furthermore, I'd really like to have a comprehensive update guide as I don't understand why some of these changes where neccassary. (https://github.com/nodejs/nan/issues/402)

**Comment by kkoopa** [comment]
Yeah, it has not been updated. The main thing that needs changing is the replacement of Nan prefix with Nan namespace, but I don't really feel like doing it. I'll merge a PR if someone wants to update it.

You can try this bash script instead https://gist.github.com/thlorenz/7e9d8ad15566c99fd116
